README.md

```diff
@@ -173,6 +173,13 @@ uv run pre-commit install
 uv run python -m normcap
 ```
 
+## Contribute to UI translations
+
+Please use [Weblate](https://hosted.weblate.org/projects/normcap/ui/) to complement or
+correct text for existing language as well as for adding new languages.
+
+(If you prefer to not use Weblate, you can also [do it manually](./normcap/resources/locales/README.md), but be aware, that this more more cumbersome.)
```

Comment:
A typo snuck in here, unless you really wanted to emphasize more 😇

Reply:
Thanks!

Reply:
Fixed in [4f3a1e5](https://github.com/dynobo/normcap/pull/677/commits/4f3a1e5f9c210f2d2f1ffe9eb18aed77d5ca2ddb)